src/python/services/queue-service/queue_service.py

```diff
@@ -0,0 +1,63 @@
+from azure.identity import AzureCliCredential
+from azure.storage.queue import QueueServiceClient
+from azure.ai.formrecognizer import FormRecognizerClient
+from azure.ai.textanalytics import TextAnalyticsClient
+from azure.cosmos import CosmosClient
+
+import requests
```

Comment:
nit: Group imports in the following order

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

Reply:
done, I think